src/types/validation-result.ts

```diff
@@ -1,9 +1,9 @@
 import type { DatasetIssues } from '../issues/datasetIssues.ts'
 
 export interface SubjectMetadata {
-  participantId: string
-  age: number
-  sex: string
+  participantId?: string
```

Comment:
Can this ever be empty?

```suggestion
  participantId: string
```